### PR TITLE
the css of home cards has been edited

### PR DESCRIPTION
**Pull request by hilalguvenc**

### CODE REVIEW

File: src/pages/Home/style.module.scss [hunk 1]
```diff
@@ -107,8 +107,142 @@
           gap: 30px;
           overflow-x: scroll;
           padding: 20px;
+          justify-content: center;
```
Comment (mehmetsefabalik) [inline src/pages/Home/style.module.scss:110]: why did you add it here?

Comment (hilalguvenc) [reply src/pages/Home/style.module.scss:110]: because I want the cards to be in the center of the screen
